test/unit-tests/ima_test.js

```diff
@@ -1839,4 +1839,20 @@ describe('ad_manager_ima', function()
       expect(_.isEmpty(sdkAdEventParams)).to.be(false);
     }
   });
+
+  it('IMA ad preloading: testing page level param', function()
```

Comment:
Do we have unit tests for the preloading (early ad request) functionality already? If not, can we add some?